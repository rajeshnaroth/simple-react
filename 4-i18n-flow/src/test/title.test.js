import React from "react"
import renderer from "react-test-renderer"

import Title from "../js/app/header/title"

test("Title gets created", () => {
  const component = renderer.create(<Title message="Hello World" />)
  expect(component.toJSON()).toMatchSnapshot()
})
