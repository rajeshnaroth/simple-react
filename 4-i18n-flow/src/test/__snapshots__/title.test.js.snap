// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title gets created 1`] = `
<section>
  <h1 />
  <h2>
    Hello World
  </h2>
</section>
`;
